tests/FailTest.cmake

```diff
@@ -0,0 +1,37 @@
+if(WIN32)
+    set(MRDOX_FT_RUNNER ${CMAKE_CURRENT_LIST_DIR}/fail_test.bat)
+else()
+    set(MRDOX_FT_RUNNER ${CMAKE_CURRENT_LIST_DIR}/fail_test.sh)
```

Comment:
dafuq is "CMAKE_CURRENT_LIST_DIR" ? what is a listfile?

Reply:
The directory of the current file